spinalcordtoolbox/scripts/sct_maths.py

```diff
@@ -45,7 +45,8 @@ def is_float(element):
         for val in values:
             # Case 1: Argument is a float
             if is_float(val):
-                data_out.append(np.full_like(data_in, float(val)))
+                shape = data_in.shape[:-1] if namespace.volumewise else data_in.shape
+                data_out.append(np.ones(shape) * float(val))
```

Comment:
Since the `DataOrScalarArgument` custom action parses `-i` to get the shape of the input, this gets a bit awkward with `-volumewise`...

Reply:
This is no longer an issue, since we no longer use `DataOrScalarArgument`.